### PR TITLE
[runSofa] ADD possibility to jump to source/instanciation of selected component

### PR DESCRIPTION
**Pull request by damienmarchal**
Two new menu entry are added in the SceneGraph view to jump to the source file location corresponding to the selected object.

This PR is based on #1012 to get the file location. 
By default qtcreator will be used  to open the file. 
But this can be changed by editting the config file saved by the QSetting mecanism. 

NB: The "Go To implementation" really make sense when using STLIB as it allows 
to jump where a prefab is implemented.  It would be nice if we could do the same for c++ 
but this is of lesser interest ...because we need to compile sofa to see a change :) 

EDIT: small video show how it works 
[video](http://www.youtube.com/watch?v=zzOY-Lt0vgM)
______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by guparan** [comment]
Thanks for this PR @damienmarchal 
I suggest adding a fallback to `QDesktopServices::openUrl(QUrl::fromLocalFile(filename));` to let the OS choose an editor (no line jump though).

**Comment by damienmarchal** [comment]
Hi Guillaume,

I hesitated to do what you suggest...but I concluded that it be better to have people to configure the feature properly instead of living with a partially working approach (no line jump). 

But...maybe with a big msg_warning() <<
 "You didnt configured your editor properly...falling back to a shitty solution without line jump"...  
 until properly configured it would be ok :)  

Can you make it ?  I'm focus on something else and have not time until tomorrow.

**Comment by guparan** [comment]
Here you go :-)

**Comment by damienmarchal** [comment]
I disagree with the "Find ..." renaming this is not a search operation (In GUI Find has a totally different meaning). 

So Go To was much better :) 
Alternative may be variation of:
"Open in external editor.."
"Open definion in external editor..."

**Comment by damienmarchal** [comment]
@guparan could you re-fix the name you changed pleased :)

**Comment by damienmarchal** [comment]
Ok after review... I closed  #1012 and updated only this one as it contains the all the commit history..

**Review by epernod — APPROVED** [review]

File: applications/sofa/gui/qt/RealGUI.cpp [hunk 1]
```diff
@@ -150,7 +150,11 @@ class QSOFAApplication : public QApplication
 public:
     QSOFAApplication(int &argc, char ** argv)
         : QApplication(argc,argv)
-    { }
+    {
+        QCoreApplication::setOrganizationName("Sofa Consortium");
```
Comment (epernod) [inline applications/sofa/gui/qt/RealGUI.cpp:154]: sure about those info?

Comment (epernod) [reply applications/sofa/gui/qt/RealGUI.cpp:154]: @hugtalbot @guparan

**Pull request merged**